.github/workflows/CI.yml

```diff
@@ -8,11 +8,10 @@ on:
       - "*"
 
 jobs:
-
   Build-ad:
     strategy:
       matrix:
-        java: [14]
+        java: [11, 14]
```

Comment:
All plugins and OpenSearch core should make sure Java 11 works for 1.3 release?

Reply:
Yes

Reply:
11 is the default used by bundle build / integ tests